src/nvim/ex_cmds.lua

```diff
@@ -38,400 +38,544 @@ module.flags = {
   PREVIEW = PREVIEW,
 }
 
+--- See `src/nvim/ex_cmds_defs.h` for the meaning of each field
+--- @class vim.ExCmd
+--- @field command string name of Ex command, e.g., 'edit'
+--- @field short_command string shorter name without the optional part, e.g., 'e' for :e[dit].
+--- @field desc string a brief description, see |ex-cmd-index|
+--- @field enum? string
+--- @field flags integer
+--- @field addr_type string
+--- @field func string
+--- @field removed? boolean deprecated or removed in neovim, should not appear in the docs
+
 -- The following table is described in ex_cmds_defs.h file.
+--- @type vim.ExCmd[]
 module.cmds = {
   {
     command = 'append',
+    short_command = 'a',
```

Comment:
Naming matters, because eventually we may want to expose this file as data in the shipped runtime files. We use `shortname` in `nvim_get_option_info2` so let's use that here.

Later we might want to change `command` to `name`, idk.
```suggestion
    shortname = 'a',
```